test/convnets.jl

```diff
@@ -215,48 +231,39 @@ GC.gc()
         end
         @test gradtest(m, x_224)
     end
-    GC.safepoint()
-    GC.gc()
+    _gc()
     @testset "MobileNetv3" verbose = true begin
         @testset for mode in [:small, :large]
             m = MobileNetv3(mode)
-
             @test size(m(x_224)) == (1000, 1)
             if (MobileNetv3, mode) in PRETRAINED_MODELS
                 @test acctest(MobileNetv3(mode; pretrain = true))
             else
                 @test_throws ArgumentError MobileNetv3(mode; pretrain = true)
             end
             @test gradtest(m, x_224)
+            _gc()
         end
     end
 end
 
-GC.safepoint()
-GC.gc()
 
 @testset "ConvNeXt" verbose = true begin
     @testset for mode in [:small, :base] #, :large # :tiny, #, :xlarge]
         @testset for drop_path_rate in [0.0, 0.5]
             m = ConvNeXt(mode; drop_path_rate)
             @test size(m(x_224)) == (1000, 1)
             @test gradtest(m, x_224)
-            GC.safepoint()
-            GC.gc()
+            _gc()
         end
     end
 end
 
-GC.safepoint()
```

Comment:
Were these not helping?

Reply:
It's the same thing, since that's the last operation inside the loop. So that's just a redundant GC cal